shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1032,12 +1146,25 @@ - (void)setEditableTransform:(NSArray*)matrix {
 // candidates view for multi-stage input methods (e.g., Japanese) when using a
 // physical keyboard.
 
+- (void)setSelectionRects:(NSArray*)rects {
+  NSMutableArray* rectsAsRect = [[NSMutableArray alloc] initWithCapacity:[rects count]];
+  for (NSUInteger i = 0; i < [rects count]; i++) {
+    NSArray* rect = rects[i];
+    [rectsAsRect addObject:@[
+      [NSNumber numberWithFloat:[rect[0] floatValue]],
```

Comment:
It looks like you are unboxing and boxing a float value here, any reason why?

Reply:
This is probably the result of a few iterations of fumbling around with some of the various issues I've encountered – it seems to work fine without it, though, so I'll take that out